tests/datasource/data_connector/test_configured_asset_azure_data_connector.py

```diff
@@ -28,6 +28,10 @@
     )
 
 
+# module level markers
+pytestmark = [pytest.mark.external_cloud_client]
```

Comment:
I think these can all be marked with `big` or `unit` if they don't actually hit an external endpoint (eg use mock endpoints). Otherwise they should be `azure` or something to indicate which cloud client (and which creds) they should be using.
It looks like a lot of these tests use mocked endpoints?